example/lib/pages/debouncing_tile_update_transformer.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:flutter_map_example/widgets/drawer/menu_drawer.dart';
+import 'package:flutter_map_example/widgets/notice_banner.dart';
+import 'package:latlong2/latlong.dart';
+
+class DebouncingTileUpdateTransformerPage extends StatefulWidget {
+  static const String route = '/debouncing_tile_update_transformer_page';
+
+  const DebouncingTileUpdateTransformerPage({super.key});
+
+  @override
+  State<DebouncingTileUpdateTransformerPage> createState() =>
+      _DebouncingTileUpdateTransformerPageState();
+}
+
+class _DebouncingTileUpdateTransformerPageState
+    extends State<DebouncingTileUpdateTransformerPage> {
+  int changeEndKeyRefresher = 0;
+  double durationInMilliseconds = 20;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Debouncing Tile Update Transformer')),
+      drawer: const MenuDrawer(DebouncingTileUpdateTransformerPage.route),
+      body: Column(
+        children: [
+          const NoticeBanner.recommendation(
+            text:
+                'This TileUpdateTransformer debounces TileUpdateEvents so they '
+                "don't occur too frequently, which can improve performance and "
+                'reduce tile requests',
+            url:
+                'https://docs.fleaflet.dev/layers/tile-layer#tile-update-transformers',
+            sizeTransition: 1200,
+          ),
+          const NoticeBanner.warning(
+            text: 'However, this does lead to reduced UX, as tiles will not be '
+                'loaded during long movements or animations, resulting in the '
+                'background grey breaking the illusion of a seamless map.',
+            url:
+                'https://docs.fleaflet.dev/layers/tile-layer#tile-update-transformers',
+            sizeTransition: 1360,
+          ),
+          Expanded(
+            child: Stack(
+              children: [
+                FlutterMap(
+                  options: MapOptions(
+                    initialCenter: const LatLng(51.5, -0.09),
+                    initialZoom: 5,
+                    cameraConstraint: CameraConstraint.contain(
+                      bounds: LatLngBounds(
+                        const LatLng(-90, -180),
+                        const LatLng(90, 180),
+                      ),
+                    ),
+                  ),
+                  children: [
+                    TileLayer(
+                      key: ValueKey(changeEndKeyRefresher),
```

Comment:
See https://github.com/fleaflet/flutter_map/blob/debouncing-tile-update-transformer/lib/src/layer/tile_layer/tile_layer.dart#L211.
I had a look at changing this, but it would require re-initialising the tile event stream often. This caused some issues. Happy to look into changing this again if it's important.

Reply:
How about some simple change like this?
```dart
key: ValueKey('TileLayer-$changeEndKeyRefresher')
```